src/expo.ts

```diff
@@ -172,6 +242,16 @@ export function projectQR(project: ProjectInfo, channel?: string): string {
   return url.toString();
 }
 
+export function createEasQr(updateId: string) {
+  assert(updateId, 'Could not create a QR code for project without the updateId');
+  const url = new URL('https://qr.expo.dev/eas-update');
```

Comment:
I suggest to rename this as:
```
const searchParams = new URL(...).searchParams
```
and then you can change the following lines as:
```
searchParams.append(....)
```

Reply:
If I do that, we cannot access the whole URL, only the `searchParams`